cli/pkg/healthcheck/healthcheck.go

```diff
@@ -130,13 +130,24 @@ func (hc *HealthChecker) globalChecks() []healthcheck.Checker {
 				}
 				return checkLabel(sa.GetLabels(), k8s.PartOfKey, k8s.PartOfVal)
 			}),
-		*healthcheck.NewChecker("buoyant-cloud-id Secret exists").
+		*healthcheck.NewChecker("agent-metadata ConfigMap exists").
+			Fatal().
+			WithCheck(func(ctx context.Context) error {
+				cm, err := hc.k8s.ConfigMap(ctx)
+				if err != nil {
+					return err
+				}
+
+				return checkLabel(cm.GetLabels(), k8s.PartOfKey, k8s.PartOfVal)
+			}),
+		*healthcheck.NewChecker("buoyant-cloud-org-credentials Secret exists").
```

Comment:
while we're modify check, can you move these two checks to immediately after `buoyant-cloud Namespace has correct labels`? it's surprising to see these checks in between `buoyant-cloud-agent` checks.
```
√ linkerd-buoyant can determine the latest version
√ linkerd-buoyant cli is up-to-date
√ buoyant-cloud Namespace exists
√ buoyant-cloud Namespace has correct labels
√ buoyant-cloud-agent ClusterRole exists
√ buoyant-cloud-agent ClusterRoleBinding exists
√ buoyant-cloud-agent ServiceAccount exists
√ agent-metadata ConfigMap exists
√ buoyant-cloud-org-credentials Secret exists
√ buoyant-cloud-agent Deployment exists
...
```